internal/storage/image.go

```diff
@@ -859,13 +915,21 @@ func GetImageService(ctx context.Context, store storage.Store, serverConfig *con
 	if serverConfig.PauseImage != "" {
 		serverConfig.PinnedImages = append(serverConfig.PinnedImages, serverConfig.PauseImage)
 	}
+	const (
+		maxCacheItems                     = 1000
+		defaultCacheTimeout time.Duration = 24 * time.Hour
+	)
 	is := &imageService{
 		lookup:               ils,
 		store:                store,
 		imageCache:           make(map[string]imageCacheItem),
 		ctx:                  ctx,
 		config:               serverConfig,
 		regexForPinnedImages: CompileRegexpsForPinnedImages(serverConfig.PinnedImages),
+		policyContexts: ttlcache.New(
```

Comment:
It might be useful for this to interact with CRI-O’s existing “trigger a reload” features.

Reply:
Yep, done :+1: